tests/test_client.py

```diff
@@ -3,6 +3,11 @@
 from typing import Any
 from unittest.mock import AsyncMock, Mock
 
+try:
+    from asyncio import timeout
+except ImportError:
```

Comment:
These tests pass, but it's a red herring - this test doesn't exercise timeouts in the client, since they're plugged in at `client.expect` (which isn't tested directly).